src/test/shared/filesystem.test.ts

```diff
@@ -52,29 +51,29 @@ describe('filesystem', () => {
         })
 
         it('accepts Buffer-based paths', async () => {
-            await filesystem.writeFileAsync(filePath, 'Hello, World!', 'utf8')
+            await writeFile(filePath, 'Hello, World!', 'utf8')
             const buffer: Buffer = Buffer.from(filePath)
 
-            assert.doesNotThrow(async () => await filesystem.accessAsync(buffer))
+            assert.doesNotThrow(async () => await access(buffer))
         })
     })
 
     describe('mkdirAsync', () => {
```

Comment:
Nit: Update the names in `describe` to match the new names.

Reply:
✔
Oh ya, the tests needed not just renaming, but a bit of fat trimming.